tests/test_forward.py

```diff
@@ -168,57 +168,57 @@ def test_retina_ghm_forward():
                 batch_results.append(result)
 
 
-# def test_cascade_forward():
-#     try:
-#         from torchvision import _C as C  # NOQA
-#     except ImportError:
-#         import pytest
-#         raise pytest.skip('requires torchvision on cpu')
-#
-#     model, train_cfg, test_cfg = _get_detector_cfg(
-#         'cascade_rcnn_r50_fpn_1x.py')
-#     model['pretrained'] = None
-#     # torchvision roi align supports CPU
-#     model['bbox_roi_extractor']['roi_layer']['use_torchvision'] = True
-#
-#     from mmdet.models import build_detector
-#     detector = build_detector(model, train_cfg=train_cfg, test_cfg=test_cfg)
-#
-#     input_shape = (1, 3, 256, 256)
-#
-#     # Test forward train with a non-empty truth batch
-#     mm_inputs = _demo_mm_inputs(input_shape, num_items=[10])
-#     imgs = mm_inputs.pop('imgs')
-#     img_metas = mm_inputs.pop('img_metas')
-#     gt_bboxes = mm_inputs['gt_bboxes']
-#     gt_labels = mm_inputs['gt_labels']
-#     losses = detector.forward(
-#         imgs,
-#         img_metas,
-#         gt_bboxes=gt_bboxes,
-#         gt_labels=gt_labels,
-#         return_loss=True)
-#     assert isinstance(losses, dict)
-#     from mmdet.apis.train import parse_losses
-#     total_loss = float(parse_losses(losses)[0].item())
-#     assert total_loss > 0
-#
-#     # Test forward train with an empty truth batch
-#     mm_inputs = _demo_mm_inputs(input_shape, num_items=[0])
-#     imgs = mm_inputs.pop('imgs')
-#     img_metas = mm_inputs.pop('img_metas')
-#     gt_bboxes = mm_inputs['gt_bboxes']
-#     gt_labels = mm_inputs['gt_labels']
-#     losses = detector.forward(
-#         imgs,
-#         img_metas,
-#         gt_bboxes=gt_bboxes,
-#         gt_labels=gt_labels,
-#         return_loss=True)
-#     assert isinstance(losses, dict)
-#     from mmdet.apis.train import parse_losses
-#     total_loss = float(parse_losses(losses)[0].item())
-#     assert total_loss > 0
+def test_cascade_forward():
+    try:
```

Comment:
could you remove this try..except block in order not to skip test
please to it in other skipped tests.

looks like it does not work because different torchvision

locally I successfully run two other tests